Brownie-Points.md

```diff
@@ -2,12 +2,11 @@ The number beside your picture on Free Code Camp tells you how many Brownie Poin
 
 ![A user profile picture next to a with Brownie Points score](http://i.imgur.com/SsvbkDH.png)
 
-There are three ways you can get Brownie Points:
+There are two ways you can get Brownie Points:
 
 1. Complete challenges - you get one point per challenge you complete    
-1. Post relevant links on [Camper News](Camper-News) - you get 1 point for posting a link, and another point for each upvote your link gets    
 1. Help other campers in chat - each time you help another camper and they thank you (by typing "thanks @yourname"), you will get a point    

```

Comment:
> ----
<p align="center">Need any help?<br>Read <a href="https://github.com/FreeCodeCamp/wiki/blob/master/.github/CONTRIBUTING.md"><b>CONTRIBUTING</b></a> Guidelines<br>Chat with us in <kbd><a href="https://gitter.im/FreeCodeCamp/Wiki"><img src="http://i.imgur.com/ThSWa6Y.png?2"> <b>FreeCodeCamp/Wiki</b></a></kbd></p>

> ----